components/autocomplete/events.md

```diff
@@ -121,32 +120,34 @@ You can use the he `OnRead` event to provide data to the component according to
 
 You can also call remote data through async operations.
 
+When using `OnRead`, make sure to set `TItem` and `TValue`.
```

Comment:
It looks like the Autocomplete does not have `TValue`. Am I missing something?

Reply:
Yes, `TValue` is always a `String`.